### DISCUSSION TITLE
Why is there ability to create new ledger, but not the ability to view them in unision?

### DISCUSSION THREAD

**Discussion by avinashkanaujiya** [ananthakumaran/paisa#34] [Q&A] [answered]
Title explains it.

**Comment by ananthakumaran** [comment]
I don't get the question. You might have to explain in more detail. What do you mean by view them in union?

**Comment by avinashkanaujiya** [comment]
I meant, that If I create a new ledger named expense.ledger, transactions from that is not being displayed in the UI, only from main.ledger, is it supposed to happen or am I doing something wrong?

Also, there is no ability to delete the ledgers.

**Reply by ananthakumaran** [reply, marked as answer]
> I meant, that If I create a new ledger named expense.ledger, transactions from that is not being displayed in the UI, only from main.ledger, is it supposed to happen or am I doing something wrong?

you have to include the file from main.ledger

```
include investments.ledger
```

> Also, there is no ability to delete the ledgers.

I will add it. I need to figure out if it can be done safely (none of the other ledger including this file etc)

**Comment by brasilikum** [comment]
@ananthakumaran 
I think I share the same confusion as OP.

While it works to include the ledger, to me it's confusing that I need to jump back to main.ledger to see the full picture of the resume on the right.

<img width="600" alt="image" src="https://github.com/ananthakumaran/paisa/assets/3106990/3f0cb1a2-1f99-4624-bd87-254ba1b28a7b">

<img width="1237" alt="image" src="https://github.com/ananthakumaran/paisa/assets/3106990/ce91cb07-340c-4876-8f3c-0e638e1f300c">
<img width="1231" alt="image" src="https://github.com/ananthakumaran/paisa/assets/3106990/211215cb-1327-4e8e-90f8-dbf098287dd6">

I think this is on purpose, but to me personally, it makes more sense that the resume on the right would be consistent between all files connected by being imported.
I assume this is harder than it seems, as a ledger can be included in more than one main file.

Edit: add main file to show structure

**Reply by ananthakumaran** [reply]
OK, I think I will throw in how I use it personally. I create a ledger file for each credit card bill, this will quickly let me know if my import has worked correctly (the balance should match the bill balance). If I show the main file, then it's not useful for me, since I am not interested in the full total, but the single file total.

I believe OP's confusion stems from not knowing the `include` feature (which I need to document).
